src/mobile/components/Navigation.jsx

```diff
@@ -0,0 +1,84 @@
+import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
+import SearchScreen from './SearchScreen';
+import HomeScreen from './HomeScreen';
+import SignUpScreen from './SignUpScreen';
+import ContactScreen from './ContactScreen';
+import { View } from 'react-native';
+
+import { MaterialIcons, MaterialCommunityIcons, Entypo, EvilIcons } from '@expo/vector-icons';
+
+const Tab = createBottomTabNavigator();
+
+const Navigation = () => {
+  return (
+    <Tab.Navigator
+      screenOptions={{
+        tabBarShowLabel: false,
+        tabBarStyle: [
+          {
+            display: 'flex',
+          },
+          null,
+        ],
+      }}
+    >
+      <Tab.Screen
+        name="Search"
+        component={SearchScreen}
+        options={{
+          tabBarIcon: () => (
+            <View style={{ alignItems: 'center', justifyContent: 'center' }}>
+              <EvilIcons name="search" size={24} color="#121D59" />
+            </View>
+          ),
+        }}
+      />
+      <Tab.Screen
+        name="Home"
+        component={HomeScreen}
+        options={{
+          tabBarIcon: () => (
+            <View style={{ alignItems: 'center', justifyContent: 'center' }}>
+              <Entypo name="home" size={24} color="#121D59" />
+            </View>
+          ),
+        }}
+      />
+      <Tab.Screen
+        name="Contact"
+        component={ContactScreen}
+        options={{
+          tabBarIcon: () => (
+            <View style={{ alignItems: 'center', justifyContent: 'center' }}>
+              <MaterialIcons name="contact-support" size={24} color="#121D59" />
+            </View>
+          ),
+        }}
+      />
+      <Tab.Screen
+        name="SignUp"
+        component={SignUpScreen}
+        options={{
+          tabBarIcon: () => (
+            <View style={{ alignItems: 'center', justifyContent: 'center' }}>
+              <MaterialIcons name="exit-to-app" size={24} color="#121D59" />
+            </View>
+          ),
+        }}
+      />
+      <Tab.Screen
+        name="theme"
+        component={HomeScreen}
+        options={{
+          tabBarIcon: () => (
+            <View style={{ alignItems: 'center', justifyContent: 'center' }}>
+              <MaterialCommunityIcons name="theme-light-dark" size={24} color="#121D59" />
```

Comment:
I'm not sure about this one.  What about this `mdiBrightness4`?

<img width="431" alt="Screen Shot 2022-02-03 at 7 53 26 PM" src="https://user-images.githubusercontent.com/427398/152454329-265532ff-3be4-4977-9949-9f673a65457d.png">

Reply:
I am using the available icons that Expo gives us, so let me double check. I was so surprised that I did not have to install any libraries to import the icons.